glue/clients/util.py

```diff
@@ -124,13 +128,23 @@ def visible_limits(artists, axis):
     if data.size == 0:
         return
 
-    data = data[np.isfinite(data)]
-    if data.size == 0:
-        return
+    if isinstance(data[0], np.datetime64) \
```

Comment:
Maybe this test could be `if isinstance(data[0], (np.datetime64, datetime.date))`